stratego/lib/stratego/NodeServer/node_server.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Stratego.NodeServer do
```

Comment:
Why can't the player node simply call the server directly: once it has the pid of the server, then calls to it would just work.